.nvmrc

```diff
@@ -1 +1 @@
-10.5.0
+10.6.0
```

Comment:
Uhhh, no. That's not a version we use.

Reply:
If you're going to upgrade then go to `12.13.0` (which is the next version we'll be using).

Reply:
yolo